backend/app/notify/email.go

```diff
@@ -227,47 +239,105 @@ func NewEmail(emailParams EmailParams, smtpParams SmtpParams) (*Email, error) {
 	return &res, err
 }
 
-// Send email about reply to Request.Email if it's set, otherwise do nothing and return nil, thread safe
-// do not returns sending error, only following:
-// 1. (likely impossible) template execution error from email message creation from Request
-// 2. message dropped without sending in case of closed ctx
+// Send email about comment reply to Request.Email if it's set,
+// also sends email to site administrator if appropriate option is set.
+// Thread safe
 func (e *Email) Send(ctx context.Context, req Request) (err error) {
-	if req.Email == "" {
-		// this means we can't send this request via Email
-		return nil
-	}
 	select {
 	case <-ctx.Done():
 		return errors.Errorf("sending message to %q aborted due to canceled context", req.Email)
 	default:
 	}
+
+	var emails []emailMessage
+
+	email, err := e.createUserEmail(req)
```

Comment:
I think i complained about it before, but every time I see "email" in either param of var it takes mental efforts to figure what this was not email address but email message. Calling it emailMsgs will help